style.css

```diff
@@ -0,0 +1,21 @@
+body {
+    background-color: blueviolet;
+}
+
+
+#survey-form{
+
```

Comment:
Nice job on making use of github inline comment to point out the block of code with the issue😉, also your comments are polite👍. kindly add a reason why this change is suggested👍. remember we were taught in our lessons on [how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) to Remember to explain why you request a change - it might be obvious for you but not for someone else..  


OPTIONALLY, You can reference the previous comment for the reason why you are suggesting the change. since they are addressng same issue👍